js/interface.js

```diff
@@ -441,6 +432,52 @@
     });
   }
 
+  /**
+   * Function that deside should we run intiLinkProvider function or not
+   * @param {int} menuItemId - an id of the menu item we should check
+   * @returns {void}
+  */
+  function checkIsProviderInited(menuItemId) {
```

Comment:
let's rename to `checkIfLinkProviderInitialized`

Reply:
@squallstar renamed